apps/builder/app/builder/sidebar-left/help-popover.tsx

```diff
@@ -45,29 +45,33 @@ export const HelpPopover = ({
             formAction="https://www.youtube.com/playlist"
             name="list"
             value="PL4vVqpngzeT4sDlanyPe99dYl8BgUYCac"
-            prefix={<Youtube1cIcon />}
+            prefix={<Youtube1cIcon fill="currentColor" />}
```

Comment:
I guess it will be in a lot of places

Reply:
no, only in those filled icons like the brand icons, 999.9% are stroke icons